clients/gax/test/gax/connection_test.exs

```diff
@@ -80,4 +80,66 @@ defmodule Gax.ConnectionTest do
     body = %Tesla.Multipart{} = Keyword.get(request, :body)
     assert 4 == length(body.parts)
   end
+
+  test "creates api client header without library version" do
+    request =
+      Request.new()
+      |> Connection.build_request()
+
+    elixir_version = System.version()
+    gax_version = Application.spec(:google_gax, :vsn)
+
+    assert [{"x-goog-api-client", "gl-elixir/#{elixir_version} gax/#{gax_version} gdcl/"}] ==
```

Comment:
Is the empty version here valid or should it be left out?

Reply:
It's valid and intentional.